sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-2021-2022.bpmn

```diff
@@ -673,6 +657,136 @@
       <bpmn:incoming>Flow_0jaycgo</bpmn:incoming>
       <bpmn:outgoing>Flow_114jf7v</bpmn:outgoing>
     </bpmn:intermediateThrowEvent>
+    <bpmn:exclusiveGateway id="Gateway_1o496q2">
+      <bpmn:incoming>Flow_0kk6ogk</bpmn:incoming>
+      <bpmn:outgoing>Flow_0201yrq</bpmn:outgoing>
+      <bpmn:outgoing>Flow_04xv6y2</bpmn:outgoing>
+      <bpmn:outgoing>Flow_1ibgfq9</bpmn:outgoing>
+    </bpmn:exclusiveGateway>
+    <bpmn:intermediateThrowEvent id="Event_15d88qb" name="Calculate Total Transportation Allowance">
+      <bpmn:extensionElements>
+        <zeebe:ioMapping>
+          <zeebe:output source="=if offeringDelivered = &#34;onsite&#34;&#10;then&#10;  dmnPartTimeProgramYearMaximums.limitTransportationAllowance * offeringWeeks&#10;else&#10;  0" target="calculatedDataTotalTransportationAllowance" />
+        </zeebe:ioMapping>
+      </bpmn:extensionElements>
+      <bpmn:incoming>Flow_0201yrq</bpmn:incoming>
+      <bpmn:outgoing>Flow_0f31yl5</bpmn:outgoing>
+    </bpmn:intermediateThrowEvent>
+    <bpmn:exclusiveGateway id="Gateway_02qbvrn">
+      <bpmn:incoming>Flow_0f31yl5</bpmn:incoming>
+      <bpmn:incoming>Flow_0ce608y</bpmn:incoming>
+      <bpmn:incoming>Flow_0sezwqd</bpmn:incoming>
+      <bpmn:outgoing>Flow_0qgamw3</bpmn:outgoing>
+    </bpmn:exclusiveGateway>
+    <bpmn:intermediateThrowEvent id="Event_1lqekii" name="Calculate Total Transportation Allowance">
+      <bpmn:extensionElements>
+        <zeebe:ioMapping>
+          <zeebe:output source="=min(&#10;  studentDataAdditionalTransportCost&#10;  ,&#10;  if (studentDataAdditionalTransportPlacement)&#10;  then&#10;    dmnPartTimeProgramYearMaximums.limitAdditionalTransportationAllowance - dmnPartTimeProgramYearMaximums.limitAdditionalTransportationPlacement&#10;  else&#10;    dmnPartTimeProgramYearMaximums.limitAdditionalTransportationAllowance&#10;)" target="calculatedDataAdditionalWeeklyTransportationCost" />
+          <zeebe:output source="=calculatedDataAdditionalWeeklyTransportationCost * min(offeringWeeks, studentDataAdditionalTransportWeeks) * dmnPartTimeProgramYearMaximums.limitAdditionalTransportationKMReduction" target="calculatedDataAdditionalTransportationMax" />
```

Comment:
I see what are you trying to do here. But feel little difficult for a dev with less background to understand.

And also I am not sure how will feel expression evaluate `if (studentDataAdditionalTransportPlacement)` because it will have `yes|no` to it.

Please create an equivalent calculatedData variable like `calculatedDataAdditionalTransportPlacement`

![image](https://github.com/bcgov/SIMS/assets/54600590/2599b0a3-ba98-4d3d-ab80-b1eb2eaddee7)

I would say to de-clutter the logic further.

Reply:
As a suggestion, something like this. 

![image](https://github.com/bcgov/SIMS/assets/54600590/84ff140c-d11f-4550-9940-2d11b83793e5)